drivers/ieee802154/Kconfig.cc13xx_cc26xx

```diff
@@ -0,0 +1,32 @@
+# Kconfig.cc13xx_cc26xx - TI CC13xx / CC26xx IEEE 802.15.4 configuration options
+#
+# Copyright (c) 2019 Brett Witherspoon
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig IEEE802154_CC13XX_CC26XX
+	bool "TI CC13xx / CC26xx IEEE 802.15.4 driver support"
+	depends on NETWORKING
```

Comment:
This depends should not be needed as that should be already checked in other Kconfig files.

Reply:
Removed